### PR TITLE
Second Newsletter of 2020 

### PR DESCRIPTION
**Pull request by jenniferisasi**
Ready to publish our more recent news as per #1804

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [ ] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [ ] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Review by fdlaramee — DISMISSED** [review]
French reviewed; minor corrections added.

**Comment by jenniferisasi** [comment]
> French reviewed; minor corrections added.

Merci @fdlaramee! As you see by a clear mistake of "l'Historien de la programmation" (Oops) I am still using translation tools, refining things that I can see are not good (for example, gender issues in translating the word "professor"). I still hope that this is faster for you and the team than having to translate the entire thing. If at some point this is a burden, please let me know.

**Comment by fdlaramee** [comment]
@jenniferisasi Translation tools sometimes make such weird choices that it takes a while to reverse-engineer their thought processes, figure out what the original text must have been, and re-translate appropriately. From my perspective, it would probably take no more time to translate everything from scratch, but @spapastamkou and the others may think differently.

File: _posts/2020-07-01-boletin-informativo-junio20.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Hemos comenzado a trabajar en la que será nuestra cuarta publicación, *Programming Historian em português*, con su primer miembro y editor gerente Daniel Alves a bordo ya en nuestro repositorio y flujo de trabajo de GitHub. Daniel es profesor asistente en el Departamento de Historia e investigador en el Instituto de Historia Contemporánea, ambos en NOVA-FCSH, Universidade Nova de Lisboa, Portugal, donde también coordina el [Laboratorio de Humanidades Digitales](https://dhlab.fcsh.unl.pt).
+
+¡Bienvenido, bem vindo Daniel!
+
+
+
+## Miembro jubilado
+
+Lamentablemente anunciamos que nuestro Jefe Técnico y editor [Matt Lincoln ha decidido jubilarse de *Programming Historian*](https://matthewlincoln.net/2020/06/18/leaving-programming-historian.html) desde el 18 de junio. Ha sido un placer tenerlo como colega y todos hemos aprendido mucho de su arduo trabajo. Creo que toda la comunidad le está agradecida por hacer que la revista sea más accesible, multilingüe y compatible con sistemas de citado. Para nosotros, los editores, facilitó la publicación de lecciones con metadatos y validación de enlaces que señalan posibles errores antes de que podamos publicar cualquier contenido nuevo.
```
Comment (rivaquiroga) [inline _posts/2020-07-01-boletin-informativo-junio20.md]: ¿Quizás sea mejor "Creemos" en vez de "Creo"? Todo lo demás está en plural

**Review by rivaquiroga — DISMISSED** [review]

**Review by rivaquiroga — APPROVED** [review]

**Pull request merged**